api/catalog/api/utils/throttle.py

```diff
@@ -61,6 +61,10 @@ class OneThousandPerMinute(AbstractAnonRateThrottle):
     rate = "1000/min"
 
 
+class OneHundredFiftyPerMinute(AbstractAnonRateThrottle):
+    rate = "150/minute"
```

Comment:
It _mayyyy_ be a good idea (given we're kind of guessing at a good rate limit for this) to make this an environment variable and change the class have a specific name for thumbs:

```suggestion
class AnonThumbnailThrottle(AbstractAnonRateThrottle):
    rate = config("ANON_THUMBNAIL_THROTTLE_RATE", "150/minute")
```

What do you think? This way if we find that it needs to be adjusted we don't have to make a code change, cut a new release, and then redeploy, it'd just be a redeploy.

Reply:
Okay, I think I have my final suggested change here to make it configurable via the environment:

1. Rename the throttle to AnonThumbnailThrottle and add a scope
2. Add the scope to the `DEFAULT_THROTTLE_RATES` in `settings.py` and set the value to an environment variable similar to how the anon throttle rates are set

It's a minor change in the context of this PR but it puts the configuration added here more in line with the rest of our throttles.

Reply:
I applied the suggested changes but doesn't seem to work well for authenticated requests, so I will convert it to draft.